Directory.Build.rsp

```diff
@@ -0,0 +1,4 @@
+/ConsoleLoggerParameters:Verbosity=Minimal;Summary;ForceNoAlign
+/MaxCPUCount
+/NodeReuse:false
```

Comment:
Why this `NodeReuse` is false? What happens if it's `true`? Does something go wrong?

Reply:
This just tells MSBuild to spawn new `MSBuild.exe` nodes and not allow them to be re-used by another build.  This used to be a more reliable way of building.  I'm not sure if its really still needed but there's no harm either way in my opinion.